README.md

```diff
@@ -39,7 +39,7 @@
 
 # Splunk OpenTelemetry Instrumentation for Android
 
-For official documentation on the Splunk OTel Instrumentation for Android, see [Instrument Android applications for Splunk RUM](https://docs.splunk.com/Observability/gdi/get-data-in/rum/android/get-android-data-in.html).
+For official documentation on the Splunk OTel Instrumentation for Android, see [Instrument Android applications for Splunk RUM](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=rum.android.gdi).
```

Comment:
Why the quickdraw.splunk.com host? It simply redirects to the docs.splunk.com link (for me)

Reply:
Those are Splunk Help Links Manager links. We use those to facilitate mass update of links should we rearrange the docs. That way, we avoid broken links.